Makefile

```diff
@@ -1024,11 +1025,15 @@ publish-ci: image-ci
 # Docker image build for Teleport Operator
 .PHONY: image-operator-ci
 image-operator-ci:
-	make -C operator docker-build IMG=$(DOCKER_IMAGE_OPERATOR_CI):$(VERSION)
+	make -C operator docker-build IMG=$(DOCKER_IMAGE_OPERATOR_STAGING):$(VERSION)
 
 .PHONY: publish-operator-ci
 publish-operator-ci: image-operator-ci
-	docker push $(DOCKER_IMAGE_OPERATOR_CI):$(VERSION)
+	@if DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $(DOCKER_IMAGE_OPERATOR_STAGING):$(VERSION) 2>&1 >/dev/null; then\
```

Comment:
Is `DOCKER_CLI_EXPERIMENTAL=enabled` strictly necessary? Could you add a comment clarifying?

Reply:
The docker experimental commands are not enabled by default in clients <20.10. I didn't want to make an assumption so I added this flag just in case. 

I added comments in https://github.com/gravitational/teleport/pull/15275/commits/7ee8b32676858a7324edc4f376525c8e7327ee56